user_guide/feature_contexts/defining_step_definitions.rst

```diff
@@ -1,7 +1,7 @@
-Defining Reusable Actions
+Defining Step Definitions
```

Comment:
Same here. the old title was more understandable IMO. People won't know what a step definition is until they have read the chapter where we explain it